apps/web/components/NavTabs.tsx

```diff
@@ -95,5 +97,4 @@ const NavTabs: FC<NavTabProps> = ({ tabs, linkProps, ...props }) => {
     </>
   );
 };
-
```

Comment:
>  In the meantime, do you think using https://eslint.org/docs/latest/rules/padding-line-between-statements on top of prettier could help?

This could help, but not for this PR. I think this will cause even more unnecessary diffs since this isn't enforced consistently through our codebase.